cluster/addons/volumesnapshots/common-snapshot-controller/common-snapshot-controller-deployment.yaml

```diff
@@ -0,0 +1,29 @@
+# This YAML file shows how to deploy the common snapshot controller 
+
+---
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: common-snapshot-controller
```

Comment:
I would just call it "volume-snapshot-controller". Common is redundant.

Reply:
ok